library/core/src/fmt/num.rs

```diff
@@ -233,10 +241,13 @@ macro_rules! impl_Display {
 
         #[cfg(not(feature = "optimize_for_size"))]
         impl $unsigned {
-            fn _fmt(self, is_nonnegative: bool, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-                const MAX_DEC_N: usize = $unsigned::MAX.ilog(10) as usize + 1;
-                // Buffer decimals for $unsigned with right alignment.
-                let mut buf = [MaybeUninit::<u8>::uninit(); MAX_DEC_N];
+            #[doc(hidden)]
+            #[unstable(
+                feature = "fmt_internals",
+                reason = "internal routines only exposed for testing",
```

Comment:
This seems inaccurate? This is being used for non-test code -- is their precedent for that? Maybe this should be core_internals or some other internal feature?

Reply:
Bad copy-pasting, good catch. Going to fix it.

Reply:
This is still incorrect, right?

Reply:
Yes indeed. I was sure I updated it though. Oh well, sending a fix.